src/index.html

```diff
@@ -1,18 +1,55 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="utf-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The `href` attribute is pointing to an SCSS file (`styles/index.scss`). Browsers cannot directly interpret SCSS files. You need to compile your SCSS into a CSS file and link to that CSS file instead.